geograypher/meshes/meshes.py

```diff
@@ -1971,9 +1969,19 @@ def vis(
             mesh_kwargs["annotations"] = IDs_to_labels
             scalar_bar_args["n_labels"] = 0
 
+        # TODO consider if there's a better CRS to use
+        vis_mesh = self.reproject_CRS(EARTH_CENTERED_EARTH_FIXED_CRS, inplace=False)
+
+        # If camera set is provided, transform the mesh into those coordinates
+        if camera_set is not None:
+            epsg_4978_to_camera = np.linalg.inv(
+                camera_set.get_local_to_epsg_4978_transform()
+            )
+            vis_mesh.transform(epsg_4978_to_camera, inplace=True)
```

Comment:
I must be missing something, but I can't find the `transform` function in `meshes.py`. Is it defined somewhere else?